Jellyfin.Server/Migrations/JellyfinMigrationBackupAttribute.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace Jellyfin.Server.Migrations;
+
+/// <summary>
+/// Marks an <see cref="JellyfinMigrationAttribute"/> migration and instructs the <see cref="JellyfinMigrationService"/> to perform a backup.
+/// </summary>
+[AttributeUsage(System.AttributeTargets.Class, Inherited = true, AllowMultiple = true)]
+public sealed class JellyfinMigrationBackupAttribute : System.Attribute
+{
+    /// <summary>
+    /// Gets or Sets a value indicating whether a backup of the old library.db should be performed.
+    /// </summary>
+    public bool LegacyLibraryDb { get; set; }
+
+    /// <summary>
+    /// Gets or Sets a value indicating whether a backup of the old library.db should be performed.
```

Comment:
Comment same as above, should one not be difference? I'm guessing one should be library.db and the other jellyfin.db ?